### PR TITLE
Add dev environment information and small fixes

### PR DESCRIPTION
**Pull request by crypblockcode**
Updates:
- Add information about how to setup a dev environment (which options you have)
- Add info about sandbox
- Small code fixes

### CODE REVIEW

File: docs/get-started/devenv/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## 3. Run your Algorand node
+
+You can decide to **run your own [Algorand node](https://github.com/algorand/go-algorand)**, which contains the full implementation of the Algorand software. This solution is more complex to set up and is also less flexible. Unlike the sandbox, you can't throw away a network and set up a new one when you want to. Setting up an Algorand node takes much more time than setting up a local, private network using the sandbox.
+
+## Recommendation?
+
+If you only want to try out Algorand code snippets, you can get started quickly with a third-party API service. However, to continue learning about Algorand and its smart signatures/smart contracts, installing the Algorand sandbox is recommended. Also, the sandbox comes with the [indexer software](https://developer.algorand.org/docs/rest-apis/indexer/#create-publication-overlay), which runs locally so you can quickly lookup applications, assets, transactions, and accounts.
```
Comment (barnjamin) [inline docs/get-started/devenv/index.md:21]: "Also, in local private networks, the sandbox comes with..."

File: docs/get-started/devenv/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Here are steps to install the Algorand sandbox:
+
```
Comment (barnjamin) [inline docs/get-started/devenv/index.md:24]: Once you have the prerequisites detailed on the sandbox readme

**Review by barnjamin — COMMENTED** [review]
Awesome! Love the new dev environment page and thanks for the other fixes as well!

Made some comments, feel free to tweak the wording

**Comment by crypblockcode** [comment]
@barnjamin Thanks for the comments - addressed them!

**Review by barnjamin — APPROVED** [review]
LGTM, Thanks again!

**Pull request merged**